Zotero/Assets/en.lproj/Localizable.strings

```diff
@@ -291,6 +291,10 @@
 "settings.general.show_subcollections_title" = "Show Items from Subcollections";
 "settings.general.show_collection_item_counts" = "Show collection sizes";
 "settings.general.open_links_in_external_browser" = "Open links in external browser";
+"settings.general.autoempty_title" = "Delete Items in Trash";
+"settings.general.after_one_day" = "After 1 Day";
+"settings.general.never" = "Never";
+"settings.general.after_x_days" = "After %d Days";
```

Comment:
`settings.general.after_one_day` and `settings.general.after_x_days` can be implemented as following in `Localizable.stringsdict` instead:
```
<key>settings.general.after_x_days</key>
    <dict>
        <key>NSStringLocalizedFormatKey</key>
        <string>%#@after_x_days@</string>
        <key>after_x_days</key>
        <dict>
            <key>NSStringFormatSpecTypeKey</key>
            <string>NSStringPluralRuleType</string>
            <key>NSStringFormatValueTypeKey</key>
            <string>d</string>
            <key>one</key>
            <string>After 1 Day</string>
            <key>other</key>
            <string>After %d Days</string>
        </dict>
    </dict>
```

and use `L10n.Settings.General.afterXDays(1)`

Reply:
Yeah I forgot we have this implementation already, fixed in c13bb883f24e478da1b01f80a83eb975854b0c96